src/components/application/ApplicationFormPage.tsx

```diff
@@ -70,38 +70,40 @@ const ApplicationFormPage: React.FC<Props> = props => {
   };
 
   return (
-    <CommonForm
-      schema={props.formPage.jsonSchema}
-      uiSchema={props.formPage.uiSchema}
-      commonDefinitionsSchema={props.commonDefinitionsSchema}
-      formData={formData}
-      onChange={({ formData: updatedFormData }, e) => {
-        setFormData(updatedFormData);
-      }}
-      onSubmit={({ formData: submittedFormData }, e) => {
-        console.log(submittedFormData);
-        handleNextClicked();
-      }}
-    >
-      <HStack justify="space-evenly">
-        <Button
-          colorScheme="purple"
-          onClick={handlePreviousClicked}
-          disabled={!props.hasPrevPage}
-          variant="outline"
-        >
-          <ArrowBackIcon />
-          {isDesktop && <Text marginLeft="2">Back</Text>}
-        </Button>
-        <Button colorScheme="purple" onClick={handleSaveData}>
-          Save
-        </Button>
-        <Button colorScheme="purple" type="submit" variant="outline">
-          {isDesktop && <Text marginRight="2">Next</Text>}
-          <ArrowForwardIcon />
-        </Button>
-      </HStack>
-    </CommonForm>
+    <Box marginLeft="10px" marginRight="10px">
```

Comment:
You can just do `marginX` for this

Reply:
But maybe a longer term fix would be to adding margin to the base `App.tsx` so every page has a margin? You'd have to check what this would affect tho.

Reply:
Is that just adding it in the app.css then?

Reply:
Yeah could be app.css, or adding a new component in app.tsx, whatever you think is best

Reply:
I messed around with it and I think adding it to app.css or adding a component to app.tsx affects some of our other components. ie: for the "Welcome Back <name>" component, we would lose how it snaps to the ends when we minimize the page. Lmk if you know any other workarounds though.
<img width="695" alt="image" src="https://user-images.githubusercontent.com/22090159/181123543-fd3b9479-3839-4f6c-bc78-8c5a1f0913f6.png">

Reply:
Ah ok ur right good catch, I think this change is good then.